homeassistant/components/google_assistant/trait.py

```diff
@@ -358,7 +358,10 @@ class ObjectDetection(_Trait):
 
     @staticmethod
     def supported(domain, features, device_class, _) -> bool:
-        """Test if state is supported."""
+        """Test if state is supported.
+
+        Only event entities with device_class `doorbell` are supported.
```

Comment:
Feels like this will just get outdated. So let's not add that, it's easily deduced from reading the code.